src/mmw/apps/modeling/calcs.py

```diff
@@ -47,12 +48,13 @@ def point_source_pollution(geojson):
     results.
     """
     geom = GEOSGeometry(geojson, srid=4326)
-
+    in_drb = geom.within(DRB)
+    table_name = get_point_source_table(in_drb)
     sql = '''
           SELECT city, npdes_id, mgd, kgn_yr, kgp_yr
-          FROM ms_pointsource
+          FROM {0}
```

Comment:
Consider using a named parameter instead of `{0}`, such as `{datasource}`, as done previously in [`streams`](https://github.com/WikiWatershed/model-my-watershed/blob/c2eacd3383753a51badfed30190dab33e3e4e03b/src/mmw/apps/modeling/mapshed/calcs.py#L331-L343), to improve readability.